### DISCUSSION TITLE
blank screen on android 5

### DISCUSSION THREAD

**Discussion by by2agst** [quasarframework/quasar#11021] [CLI - Capacitor / Cordova modes]
I tried to build an app using quasar v2, this app can run normally on android 5+ but when I try it on android 5 it can't. and it only shows a blank screen with the error **"Unexpected reserved word"**
Kindly assist, thanks!

![2021-10-18_172019](https://user-images.githubusercontent.com/21094294/137715013-8571f4e9-3b48-423f-83ac-e7d8a9fe603c.png)
![2021-10-18_172106](https://user-images.githubusercontent.com/21094294/137715017-eb1dea8a-d32d-44a7-8737-58c7782dd661.png)
![2021-10-18_172205](https://user-images.githubusercontent.com/21094294/137715020-05c1c823-7eb4-44a6-b9b9-122db02e2379.png)
![2021-10-18_172525](https://user-images.githubusercontent.com/21094294/137715023-6500820b-c077-4363-a520-f2a27944cc8f.png)

**Comment by rstoenescu** [comment]
You have conflicts in your list. "last 8 Android versions" and "Android > 4".
One thing to keep in mind when using "last X versions" (not specifically for this scenario) doesn't refers to last Major versions, but any version (including minor or patch).

**Reply by by2agst** [reply]
Thank you for answering
Then the recommended "browserslist" to run on Android 5.0?
I tried just using "Android >= 4", but it didn't work.
